src/.env.example

```diff
@@ -0,0 +1,13 @@
+DATABASE=postgres
+DATABASE_USER=postgres
+DATABASE_PASSWORD=docker
+
+{
+  "label": "127.0.0.1",
+  "host": "127.0.0.1",
+  "user": "postgres",
+  "port": 5432,
+  "ssl": false,
+  "database": "",
+  "password": "docker"
+}
```

Comment:
.env files should be just for development ideally as they state in their docs:

![image](https://user-images.githubusercontent.com/13842678/61387386-6a129000-a88c-11e9-8c4b-6070eb4342a9.png)

Also, I never seem this JSON syntax on line 5 to 13 in dotenv files, are you sure this works?

Let your .env file be in the root of the project.

name it only .env as it's likely to be the only one you'll need.

Reply:
That's not the real .env file. that's just an example of the file I use, to facilitate the use for someone else.

From lines 5 to 13 was my docker configuration settings, I will remove in the next commit.